### ISSUE TITLE
link to classification information

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/nmdc-schema#678]

**Comment by mslarae13** [comment]
? Leave it out? Not really used... non-actionable DB field

**Comment by turbomam** [comment]
`link_class_info` values in the BBOP SQLite version of NCBI `biosample_set.xml` as of 2023-05-18

|value|count|
|-----|-----|
|not collected|432|
|not applicable|415|
|NA|114|
|http://geointa.inta.gov.ar/visor/?p=model_suelos|108|
|http://doi.org/10.1002/jpln.200521814|71|
|missing|65|
|http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups|64|
|https://www.fao.org/3/i3794en/I3794en.pdf|60|
|Chromic Haploxernt|48|
|na|19|
|doi:10.1016/j.fcr.2011.09.019|6|
|http://esdac.jrc.ec.europa.eu/resource-type/european-soil-database-maps|1|

**Comment by turbomam** [comment]
Environmental package | agriculture | soil
-- | -- | --
Structured comment name | link_class_info | link_class_info
Package item | link to classification information | link to classification information
Definition | Link to digitized soil maps or other soil classification information | Link to digitized soil maps or other soil classification information
Expected value | PMID,DOI or url | PMID,DOI or url
Value syntax | {PMID\|DOI\|URL} | {PMID}\|{DOI}\|{URL}
Example |   |  
Requirement | X | X
Preferred unit |   |  
Occurrence | 1 | 1
MIXS ID | MIXS:0000329 | MIXS:0000329

**Comment by turbomam** [comment]
It seems like `link_class_info` might function like one of the many `x_meth` fields, which state how the value for slot `x` was determined. But I can't tell what slot `link_class_info` might provide context for.

As shown above, the number of biosamples that are annotated with an informative `link_class_info` is in the low hundreds, out of 35 million. (I am defining informative as something other than a synonym for "not available" or a live web link.)

@mslarae13 I propose that we omit this field from NMDC submission templates. A longer term action could be filing an issue at https://github.com/GenomicsStandardsConsortium/mixs/issues asking for **documentation** or **examples** of this term's use. I hope that that request wouldn't be construed as invitation for open-ended discussion about the subject. I suppose it could also be useful to be put in touch with the person who requested that term in the first place.

**Comment by turbomam** [comment]
`link_class_info` has not been used for any biosamples in the NMDC production MongoDB yet

```js
db.getCollection("biosample_set").find( { link_class_info : { $exists : true } } );
```
>0

**Comment by ssarrafan** [comment]
Adding to current sprint per Mark. Need feedback from @mslarae13

**Comment by mslarae13** [comment]
>  I propose that we omit this field from NMDC submission templates. A longer term action could be filing an issue at https://github.com/GenomicsStandardsConsortium/mixs/issues asking for documentation or examples of this term's use.
-- I agree with this @turbomam

**Comment by ssarrafan** [comment]
@turbomam can this be closed now that Montana has provided feedback?

**Comment by mslarae13** [comment]
@ssarrafan  I don't think we can close it yet. Steps for resolution are remove from NMDC submission portal. & submit issue to GSC

**Comment by ssarrafan** [comment]
> @ssarrafan I don't think we can close it yet. Steps for resolution are remove from NMDC submission portal. & submit issue to GSC

Ah ok.  I thought this ticket was just to "determine if and how..."  
I'll move to the next sprint

**Comment by mslarae13** [comment]
https://github.com/GenomicsStandardsConsortium/mixs/issues/590

**Comment by turbomam** [comment]
I'll remove it in 7.6.1

**Comment by mslarae13** [comment]
@turbomam  did this get removed?

**Comment by mslarae13** [comment]
Currently manually managing. We do not ingest all MIxS slots, see https://github.com/microbiomedata/nmdc-schema/blob/main/assets/import_mixs_slots_regardless.tsv

MIxS that we pull into NMDC schema is 6.0 ..Should pull in 6.2, but some slot changes have been made (names, presence/absence)
- Will require migration
- How/ when with respect to berk? Does it affect postgres?
- There are some no impact changes. samp_collec_meth & device

**Comment by ssarrafan** [comment]
I'm moving this to the current sprint based on @mslarae13 last comment on 4/29.

**Comment by ssarrafan** [comment]
@turbomam @mslarae13 should this just be in the backlog? I can remove it from the sprint.

**Comment by ssarrafan** [comment]
Sprint over, removing from sprint.
